sources/cloudfront/realtime_log_configs.go

```diff
@@ -107,7 +107,7 @@ func NewRealtimeLogConfigsSource(client *cloudfront.Client, accountID string) *s
 			logConfigs := make([]*types.RealtimeLogConfig, 0, len(out.RealtimeLogConfigs.Items))
 
 			for _, logConfig := range out.RealtimeLogConfigs.Items {
-				logConfigs = append(logConfigs, &logConfig)
+				logConfigs = append(logConfigs, &logConfig) // nolint:exportloopref // this is resolved in go 1.22 https://github.com/kyoh86/exportloopref
```

Comment:
We're never gonna run a go version that is older than 1.22; let's add this ignore to the the default configs.